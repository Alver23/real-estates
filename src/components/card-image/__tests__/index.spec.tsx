// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import CardImage from '../index';

describe('<Footer />', () => {
  let component: ShallowWrapper;

  const props = {
    image: 'http://localhost/img.png',
    alt: 'fake image',
  };

  beforeEach(() => {
    component = shallow(<CardImage {...props} />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly', () => {
    expect(component.find('img').exists()).toBeTruthy();
  });
});
