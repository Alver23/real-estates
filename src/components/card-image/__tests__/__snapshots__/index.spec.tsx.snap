// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should save a snapshot of the component 1`] = `
<Card>
  <img
    alt="fake image"
    className="w-full border border-white | object-cover | rounded-lg"
    src="http://localhost/img.png"
  />
</Card>
`;
